### ISSUE TITLE
Keyboard/Mouse not working after it loads to the installation screen

### ISSUE THREAD

**Issue by newsera** [git-coder988/Dell-Optiplex-3040-OpenCore-Hackintosh#1]
I have a 3040 SFF and trying to make use of onboard graphics. I can't get the keyboard / mouse to work after it goes to the installation screen (Language selection). Any thoughts on this?

**Comment by newsera** [comment]
Are you also using iMacPro1,1 for Type?

**Comment by git-coder988** [comment]
> I have a 3040 SFF and trying to make use of onboard graphics. I can't get the keyboard / mouse to work after it goes to the installation screen (Language selection). Any thoughts on this?

The config.plist for this 3040 is using a dGPU.  If you want to use integrated graphics, you need to update DeviceProperties.  Follow this section in the guide:  https://dortania.github.io/OpenCore-Install-Guide/config.plist/skylake.html#deviceproperties

OR...you can copy the device properties from my 7040 config.  This config is using onboard graphics:  https://github.com/git-coder988/Dell-Optiplex-7040-OpenCore-Hackintosh

As for keyboard and mouse not working, set Kernel->Quirks->CustomSMBIOSGuid to FALSE.  Dortania guide is wrong about this one.  Keyboard and mouse don't work when set to true.

FYI, I think it'll be quicker to get up and running using my 7040 config.  Just remove IntelMausi.kext and replace with RealtekRTL8111.kext and you're good to go.  DeviceProperties is a better match.

Good luck.

**Comment by git-coder988** [comment]
> Are you also using iMacPro1,1 for Type?

Yes, iMacPro1,1.  Networking doesn't work with iMac17,1.

**Comment by newsera** [comment]
Many thanks for clarifying. I followed your recommendation of modifying DeviceProperties from the skylake guide and it was straightforward. But this time, I got stuck in Security Policy Loaded: Aple System Policy (ASP). So I set ReleaseUsbOwnership = True in config.plist and it started working. I am glad that I was able to get past the screen with a working keyboard/mouse. This was for BigSur installation. Many thanks to you once again!
